src/components/Breadcrumbs/Breadcrumb.styles.js

```diff
@@ -1,18 +1,37 @@
 import { ListUnstyled } from "../List";
 import typography from "../../theme/typography";
 import spacing from "../../theme/spacing";
+import theme from "../../theme/colorThemes";
 
 const StyledBreadcrumb = ListUnstyled.extend`
   font-weight: ${typography.weight.semiBold};
   display: flex;
   flex-flow: row;
-  color: ${({ color }) => color || "inherit"};
   ${({ childrenLen }) =>
     childrenLen &&
     `
     flex: 0 1 ${Math.floor(100 / childrenLen)}%;
   `} align-items: center;
 
+  &.breadcrumb--none {
+    color: inherit;
+  }
+
+  &.breadcrumb--light {
+    color: ${({ theme: { themeName = "tm" } = {} } = {}) =>
+      theme[themeName].white.base};
+  }
+
+  &.breadcrumb--dark {
+    color: ${({ theme: { themeName = "tm" } = {} } = {}) =>
+      theme[themeName].onyx.base};
+  }
+
+  &.breadcrumb--accent {
+    color: ${({ theme: { themeName = "tm" } = {} } = {}) =>
+      theme[themeName].primary.base};
+  }
```

Comment:
Btw, @Stan-Che added a nifty util for this common pattern: https://github.com/Ticketmaster/aurora/blob/master/src/utils/getThemeValue.js

It would be great to add the `"tm"` default in there

Reply:
good call I'll update